functions/_fzf_preview_changed_file.fish

```diff
@@ -0,0 +1,24 @@
+function _fzf_preview_changed_file
+    set -l path (string split ' ' $argv)[-1]
+    if string match -r '^\?\?' $argv >/dev/null
+        echo Untracked
+        echo
+        _fzf_preview_file $path
+    else if string match -r '^A\s\s\S' $argv >/dev/null
```

Comment:
For the regex, I get why the ^A, but how come you also have to do \s\s\S?

Reply:
to match `[AMDCRT][MCDRT] file`

I haven't tried to simplify it since it works.